src/components/UNSTABLE__Shell/PopoverAccount/index.js

```diff
@@ -0,0 +1,50 @@
+/**
+ * @file Popover account.
+ * @copyright IBM Security 2020
+ */
+
+import classnames from 'classnames';
+import { node, string } from 'prop-types';
+import React from 'react';
+
+import { appendComponentNamespace } from '../../../globals/namespace';
+
+import { namespace as profileNamespace } from '../ProfilePopover';
+
+export const namespace = appendComponentNamespace(profileNamespace, 'body');
+
+function PopoverAccount({ account, children, className, id, ...other }) {
+  return (
+    <section
+      className={classnames(namespace, `${namespace}--account`, className)}
+      {...other}
+    >
+      <div className={`${namespace}__label`}>
+        {account}: <span className="selectable-text">{id}</span>
```

Comment:
Why doesn't the `selectable-text` class name have a namespace? (Should it?)

Reply:
It should! Because we're leveraging the current implementation's styles, I wanted to avoid touching anything that may cause a noticeable breaking change - I'll need to create a task for fully consolidating the styles once we're ready to deprecate the old `Header`, `Shell`, and `Toolbar`